packages/types/src/schemas/tutorial.ts

```diff
@@ -3,6 +3,7 @@ import { webcontainerSchema } from './common.js';
 
 export const tutorialSchema = webcontainerSchema.extend({
   type: z.literal('tutorial'),
+  logoLink: z.string().optional(),
```

Comment:
Did we now decide whether to use the TK integration for configuration or a separate file? Personally I'd prefer if we decided that and maybe added support for that in this PR before we go back and forth on frontmatter configurations? I mean it's always a little cumbersome for users if they have to update and if we have folks that use the frontmatter and we switch it to a config file very soon then updating isn't as smooth even if it's a small change but if we can avoid it that'd be great.

Reply:
We haven't decided on anything yet so I haven't changed it.

My opinion? I think that property should be in a frontmatter rather than in the configuration of the TK integration or a separate configuration file.

Here's why I think so:

 * Having it in the frontmatter instead of a configuration file means that there's no need for the vite server to be restarted. Which makes sense given I'm only setting an href on an element in the UI.
 * In the last sync meeting, we've been discussing of having a `TopBar.astro` (or equivalent) that could be picked up automatically to give full control over how the top bar is configured. This would not require any configuration change, so having one for the logoLink is weird
 * We have `editor: false` which is also a frontmatter setting right now and also a UI concern so I see arguments of having `logoLink` in a config to also apply to `editor`.

Lastly, I would keep configuration at the root of the repo to be focused on stuff like port, headers, routes, authentication. Today we have:
* `enterprise` to point at a StackBlitz Enterprise instance
* `defaultRoutes` which when set to false let Astro lookup routes in `src/pages` useful when you want full customizablility.

If possible I'd like to keep it this way. YMMV.